addons/event_product/models/event_event_ticket.py

```diff
@@ -27,6 +33,17 @@ def _compute_price_reduce_taxinc(self):
             taxes = tax_ids.compute_all(event.price_reduce, event.event_id.company_id.currency_id, 1.0, product=event.product_id)
             event.price_reduce_taxinc = taxes['total_included']
 
+    def _compute_total_price_reduce_taxinc(self):
+        for event_ticket in self:
+            linked_products_taxes = 0
+            for product in event_ticket.additional_product_ids:
+                tax_ids = product.taxes_id.filtered(lambda r: r.company_id == event_ticket.event_id.company_id)
+                contextual_discount = product._get_contextual_discount()
+                product_price_reduce = (1.0 - contextual_discount) * product.lst_price
+                taxes = tax_ids.compute_all(product_price_reduce, event_ticket.event_id.company_id.currency_id, 1.0, product=product)
```

Comment:
Shouldn't we use the currency of the product instead?

Reply:
To me it make senses to apply the currency of the company of the event as everything should be compute based on that currency prior to the additional products. So I think that we need to keep every price in the same currency for the computation to make sense 🤔